coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -186,15 +186,15 @@ <h2 class="section-label">Testimonials</h2>
       <template is="dom-if" if="[[!currentUser.loggedIn]]">
         <a href="[[currentUser.loginUrl]]">
           <div class="continue-learning">
-            <a class="waves-effect waves-light btn-large" target="_blank">Start Learning</a>
+            <a class="waves-effect waves-light btn-large" target="_blank" href="[[currentUser.loginUrl]]">Start Learning</a>
```

Comment:
@Sharonsyra Is there a reason why we are repeating the href from https://github.com/andela-angene/coursebuilder-core/blob/26175dba36e93d74695ea706f0abdd323b7f0004/coursebuilder/modules/explorer/_static/components/pages/homepage.html#L187

Reply:
fixed